GitVersionConfig.yaml

```diff
@@ -1,3 +0,0 @@
-mode: ContinuousDeployment
-assembly-versioning-scheme: MajorMinorPatch 
-#next-version: 1.0.0
```

Comment:
@AArnott so, just so we are clear... Does that mean you are planning on removing the inclusion of Nerdbank.GitVersioning from this PR?  I see little point in adding it, if the general consensus is to use GitVersion.